frontend/src/components/Checkbox/index.module.css

```diff
@@ -0,0 +1,33 @@
+.label {
+  color: var(--color-black);
+  pointer-events: none;
+}
+
+.input {
+  width: var(--font-size-md);
+  height: var(--font-size-md);
```

Comment:
Since this `width` and `height` are coupled with the `width` and `height` of the `.control` element, it would be nice if you could create local variables and use those. Because a developer that comes after you might not realize that those two are coupled and might only change the size in one place, effectively breaking the feature,

Reply:
You can declare something like this at the top of the file:

```
--size-checkbox: 24px;
```

Then:

```
width: var(--size-checkbox);
height: var(--size-checkbox);
```

Reply:
You can also use the global variables if you prefer. I still don't agree with using the `font-size` one here though, but it would look something like this:

```
---size-checkbox: var(---font-size-md)
```

Then

```
width: var(--size-checkbox)
height: var(--size-checkbox)
```

Reply:
That makes sense!

Reply:
So it gives me an error unless I put it inside something. I that I could put it inside :root , is that correct?

Reply:
Putting it inside `:root` will give you access to this variable in your entire project. You only need it locally. Try adding it under the top class.